### PR TITLE
DEV-2295 Document pragma: no cover

### PR DESCRIPTION
**Pull request by njharman**
#### What's this PR do?
  - Fix test njharman forget to reenable and fix.
  - Cut pragma: no cover that aren't needed. Document the rest.
  - xfail tests for code that may be wrong rather than pragma: no cover it.
  - Add some coverage. Over 97%, nothing below 90%.

#### Why are we doing this? How does it help us?
Tested code is happy code. Ignoring code should be last resort and documented.

#### How should this be manually tested? Please include detailed step-by-step instructions.
N/A

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
No

#### Have automated unit tests been added? If not, why?
Indeed

#### How should this change be communicated to end users?
N/A

#### Are there any smells or added technical debt to note?
Added two expected fail tests in ./apps/common/context_processors.py seem maybe unused. 

#### Has this been documented? If so, where?
N/A

#### What are the relevant tickets? Add a link to any relevant ones.
[DEV-2295]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
No.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
No.

[DEV-2295]: https://news-revenue-hub.atlassian.net/browse/DEV-2295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/users/management/commands/bootstrap-review-app.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from apps.organizations.models import RevenueProgram
 
 
-class Command(BaseCommand):  # pragma: no cover
+class Command(BaseCommand):  # pragma: no cover Guess we're too lazy to test.
```
Comment (x110dc) [inline apps/users/management/commands/bootstrap-review-app.py:17]: Please change this to something more factual and less judgmental.

File: apps/users/management/commands/teardown-review-app.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from apps.contributions.utils import get_hub_stripe_api_key
 
 
-class Command(BaseCommand):  # pragma: no cover
+class Command(BaseCommand):  # pragma: no cover Guess we're too lazy to test
```
Comment (x110dc) [inline apps/users/management/commands/teardown-review-app.py:13]: Same as above. Please change.

File: apps/common/tests/test_context_processors.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from unittest import mock
+
+import pytest
+
+from apps.common.context_processors import commit_sha, sentry_dsn
+
+
+@pytest.mark.xfail(reason="Seems unused? SENTRY_DSN not in settings. TODO: delete or document")
+def test_sentry_dsn():
```
Comment (x110dc) [inline apps/common/tests/test_context_processors.py:9]: The COMMIT_SHA in cloudbuild.yaml is not related to anything in Django. I'm pretty sure neither of these are used in any way and can be removed.

**Review by x110dc — CHANGES_REQUESTED** [review]
It's okay with me if the SENTRY_DSN and COMMIT_SHA removals happen separately. Otherwise just two requests to change language in comments.

File: apps/users/management/commands/teardown-review-app.py (hunk 2 above)
Comment (njharman) [reply apps/users/management/commands/teardown-review-app.py:13]: done

**Review by x110dc — APPROVED** [review]

**Pull request merged**